components/scream/src/control/surface_coupling.hpp

```diff
@@ -35,8 +39,7 @@ class SurfaceCoupling {
   using cpl_data_ptr_type = double*;
 
   // The input grid is the one where import/export happens
-  SurfaceCoupling (const std::shared_ptr<const AbstractGrid>& grid,
-                   const FieldRepository<Real>& repo);
+  SurfaceCoupling (const field_mgr_ptr& field_mgr);
```

Comment:
The pedantic C++ programmer in me wants the `explicit` keyword here. :-) What a language.